app/assets/stylesheets/themes/oxford.sass

```diff
@@ -0,0 +1,57 @@
+@import url(https://fonts.googleapis.com/css?family=Montserrat)
+
+body.oxford
+  @mixin dark-highlight
+    background-color: #572B38
+
+  background: #2B3856
+  color: #FFFFF0
+
+  .nav > li > a:hover,
+  .nav > li > a:focus,
+  .nav .open > a,
+  .dropdown-menu
+    @include dark-highlight
+
+  .dropdown-menu > li > a:hover
+    background: #2A5749
+    color: #FFFFF0
+
+  .nav-tabs > li.active > a,
+  .nav-tabs > li.active > a:hover,
+  .nav-tabs > li.active > a:focus
+    @include dark-highlight
+    color: inherit
+    border: 1px solid #2A5749
+    border-bottom-color: transparent
+
+  h1
+    font-family: 'Montserrat', 'Helvetica Neue', Helvetica, Arial, sans-serif
+
+  a
+    color: #f9d423
+  a.btn
+    color: #FFFFF0
+  a.btn-default
+    color: #000
+
+  input, select
+    color: #000
+
+  .label-success a
+    color: #1B312F
+
+  .panel-default
+    background: transparent
+    border: none
+
+    .panel-heading
+      @include dark-highlight
+      color: #f5f5f5
+      border: none
+
+  .table-striped > tbody > tr:nth-of-type(odd)
+    @include dark-highlight
+
+  .panel-success .panel-heading
+    background-color: #464
```

Comment:
Bah. I rebased off `master` prior to starting this latest block of work and pulled in my now-merged Oxford theme feature.

:disappointed: